sys/include/net/sntp.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2016 Luminita Lazarescu <cluminita.lazarescu@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_sntp Simple Network Time Protocol
+ * @ingroup     net
+ * @brief       Simple Network Time Protocol (SNTP) implementation
+ * @{
+ *
+ * @file
+ * @brief       SNTP definitions
+ *
+ * @author      Luminita Lazarescu <cluminita.lazarescu@gmail.com>
+ * @author      Martine Lenders <m.lenders@fu-berlin.de>
+ */
+
+#ifndef SNTP_H_
+#define SNTP_H_
+
+#include <stdint.h>
+#include "net/sock/udp.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Syncronize with time server
+ *
+ * @param[in] server    The time server
+ * @param[in] timeout   Timeout for the server response
+ *
+ * @return 0 on success
+ * @return negative number on error
+ */
+int sntp_sync(sock_udp_ep_t *server, uint32_t timeout);
+
+/**
+ * @brief Get real time offset from system time
+ *
+ * @return real time offset in microseconds
```

Comment:
This is still relative to 1900-01-01 00:00 UTC I realized now, so this should be documented